spotinst/commons/consts.go

```diff
@@ -187,6 +187,8 @@ const (
 	OceanAKSNPVirtualNodeGroupNodeCountLimits    ResourceAffinity = "Ocean_AKS_NP_Virtual_Node_Group_Node_Count_Limits"
 	OceanAKSNPVirtualNodeGroupNodePoolProperties ResourceAffinity = "Ocean_AKS_NP_Virtual_Node_Group_Node_Pool_Properties"
 	OceanAKSNPVirtualNodeGroupVmSizes            ResourceAffinity = "Ocean_AKS_NP_Virtual_Node_Group_Vm_Sizes"
+	AccountAWS                                   ResourceAffinity = "Account_AWS"
+	CredentialAWS                                ResourceAffinity = "Credential_AWS"
```

Comment:
change to  CredentialsAWS = "Credentials_AWS"

Reply:
changed to CredentialsAWS = "Credentials_AWS"

Reply:
changed to CredentialsAWS = "Credentials_AWS"

Reply:
changed to CredentialsAWS = "Credentials_AWS"